src/PrestaShopBundle/Resources/config/services/bundle/bridge.yml

```diff
@@ -2,33 +2,37 @@ services:
   _defaults:
     public: true
 
+  # @todo: services naming should follow classnames
```

Comment:
Agreed, you can drop the `.core` part (PrestashopBundle is written `prestashop` in most other service definitions which is acceptable I'd say) it's still time to modify this before the final release